omero/users/cli/import.rst

```diff
@@ -48,11 +48,13 @@ Examples of options available to the import command,
         $ bin/omero import image.tif -T Dataset:id:2
         $ bin/omero import image.tif -T Dataset:name:Sample01
 
-    See :doc:`import-target` for more information on import targets.
+        See :doc:`import-target` for more information on import targets.
```

Comment:
Bad indent.  This is included verbatim so the link is broken and part of the above example.